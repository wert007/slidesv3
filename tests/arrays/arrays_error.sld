func main() {
    let a = [1, 2, 3];
    print(a[-1]);
    a[-1] = 0;
    print(a);

    let underspecified = [];
}
