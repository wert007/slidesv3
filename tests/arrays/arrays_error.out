Error in ..\tests\arrays\arrays_error.sld:3:13: Cannot convert type int to type uint.
Error in ..\tests\arrays\arrays_error.sld:4:7: Cannot convert type int to type uint.
Error in ..\tests\arrays\arrays_error.sld:7:26: Struct Array is a generic struct, but its generic type is undeclared.
